ltm/agent.py

```diff
@@ -259,56 +283,39 @@ def keywords_for_message(self, user_message, cost_cb):
         print(f"Interaction keywords: {keywords}")
         return keywords
 
-    def create_context(self, user_message, max_prompt_size, previous_interactions, cost_cb):
-        context = [make_system_message(self.system_message), make_user_message(
-            f"{str(self.now)[:-7]} ({td_format(datetime.timedelta(seconds=1))}) " + user_message)]
-        relevant_interactions = self.get_relevant_memories(user_message, cost_cb)
-
-        # Get interactions from the memories
-        for m in relevant_interactions:
-            if "trivia" in m[0].content:
```

Comment:
This statement here is incredibly useful for at-a-glance sanity checking in-progress runs. If the memories are not what you expect then you can investigate further or terminate the run. Can we wrap some of this as a debug statement?